### PR TITLE
Update information about software players and add comparison table

### PR DESCRIPTION
**Pull request by terual**
See #25

### CODE REVIEW

File: docs/players-and-controllers/softsqueeze.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# SoftSqueeze
+
+<figure markdown="span">
+  ![](assets/softsqueeze-transporter.png){ width="400" }
+  <figcaption>SoftSqueeze with the Transporter skin</figcaption>
+</figure>
+
+SoftSqueeze is a community supported software music player that allows you to play music streamed from Lyrion Music Server. Since 2009 SoftSqueeze is no longer included in the LMS distribution and is replaced by [SqueezePlay](squeezeplay.md). Although it has since been replaced, SoftSqueeze can still be useful in the present day to simulate the screens of for instance the [Squeezebox Classic](squeezebox-classic.md) and [Transporter](transporter.md). The SoftSqueeze emulator has the look and feel of the [Transporter](transporter.md), [Squeezebox Classic](squeezebox-classic.md), [Squeezebox Boom](squeezebox-boom.md), [Squeezebox 2](squeezebox2.md) and [Squeezebox 1](squeezebox1.md) hardware, and their remote controls.
+
+SoftSqueeze was written in Java by Richard Titmuss and is currently maintained by Ralph Irving.
```
Comment (michaelherger) [inline docs/players-and-controllers/softsqueeze.md:15]: You could link the two profiles here.

**Comment by michaelherger** [comment]
That's another nice one!

**Comment by terual** [comment]
Thanks for the merge! But I had assigned a review to @ralph-irving as discussed in #25 in case I had written something which would be somewhat incorrect.

**Comment by ralph-irving** [comment]
I just finished reading it.  Looks great!  Thanks for keeping in the loop.

**Comment by michaelherger** [comment]
Oops... sorry! Would be worth putting such information in the PR description, or even open it as a draft PR (there's an arrow on the creation button).

**Comment by terual** [comment]
> Oops... sorry! Would be worth putting such information in the PR description, or even open it as a draft PR (there's an arrow on the creation button).

No problem :) Will do that next time

**Comment by ralph-irving** [comment]
I'm not always quick to respond, depends on the day.
So rather than me holding things up, let me know what you'd like reviewed.
If I think there should be changes, I can comment in the commit, or raise a PR.

**Pull request merged**